### PR TITLE
Musl fix and gcc8 fixes

### PR DESCRIPTION
**Pull request by kraj**

### CODE REVIEW

File: src/ecezo/ecezo.c [hunk 1]
```diff
@@ -488,7 +488,7 @@ int ecezo_send_command(const ecezo_context dev, char *cmd, char *buffer,
     // our write buffer
     char writeBuffer[ECEZO_MAX_BUFFER_LEN];
 
-    strncpy(writeBuffer, cmd, ECEZO_MAX_BUFFER_LEN);
+    memcpy(writeBuffer, cmd, ECEZO_MAX_BUFFER_LEN-1);
     writeBuffer[ECEZO_MAX_BUFFER_LEN - 1] = 0;
```
Comment (Propanu) [inline src/ecezo/ecezo.c]: Seems redundant, you can initialize the entire buffer and remove this line.

Comment (kraj) [reply src/ecezo/ecezo.c]: Not sure if that will be efficient.

Comment (Propanu) [reply src/ecezo/ecezo.c]: Nvm, after a second look I figured this terminates the buffer correctly when `cmd >= writeBuffer`.

**Comment by Propanu** [comment]
Squashed and merged, thanks!!

**Pull request closed without merging**